.circleci/config.yml

```diff
@@ -405,9 +441,17 @@ workflows:
               only: main
           requires:
             - docs-production
+      - deploy-verdaccio:
+          requires:
+            - docs-verdaccio
       - comment:
+          name: comment-pr
           requires:
             - deploy
+      - comment:
```

Comment:
I think it's problematic for this to have the same key as the one above it? why don't they conflict?

Reply:
The `name` provided to the job handles distinguishing the two jobs even though they use the same overall `comment` job defined above https://github.com/adobe/react-spectrum/blob/f8b270c1da9f19c8d19583fefcd8f44671226506/.circleci/config.yml#L345-L357

Results in a flow like so: https://app.circleci.com/pipelines/github/adobe/react-spectrum/9885/workflows/9ca28065-70c7-4494-9f9b-639838077f23